### ISSUE TITLE
Juddery playback on 16.5.3

### ISSUE THREAD

**Issue by JFG90** [koying/SPMC#602]
Hi there been testing the newest update it works perfectly except for when i play IPTV streams the stream starts becoming Juddery every 30 seconds or so, tried playing about with other settings and still the same, I've had to revert back to 16.4.2 for now!

**Comment by JFG90** [comment]
16.4.2 plays them perfectly!

**Comment by nekromantik-zz** [comment]
how did you revert back to 16.4.2? Did you have to remove and reinstall?

**Comment by JFG90** [comment]
Just removed and reinstalled from playstore

**Comment by nekromantik-zz** [comment]
thanks

**Comment by JFG90** [comment]
No worries! I noticed everything else was perfect though! But reverted back for now!

**Comment by nekromantik-zz** [comment]
Yeah I did backup of my add ons and settings and restored on clean 16.4.2 and now no more crashes.
50% of the time I did see exit crashes but none when playing or stopping a movie.

**Comment by JFG90** [comment]
Yes i used to get crashes to when coming back out of a stream at times in exodus, unfortunately spmc has stopped working, it appears its fixed from what I've seen in 16.5.3 but the live tv streams judder hopefully its sorted on the next update!
